pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> An audio player.
+> More information: <http://qmmp.ylsoftware.com>.
+
+- Launch qmmp:
+
+`qmmp`
+
+- Start/stop the currently playing audio:
+
+`qmmp --play-pause`
+
+- Seek forwards/backward a specific amount of time (in seconds):
+
+`qmmp --seek-{{fwd|bwd}} {{time_in_seconds}}`
+
+- Play the next audio file:
+
+`qmmp --next`
+
+- Play the previous audio file:
+
+`qmmp --previous`
```

Comment:
```suggestion
- Play the next or previous audio file:

`qmmp --{{next|previous}}`
```

- This change is optional - no corresponding style guide requirement exists.
- Two consecutive examples could be joined via using `or` technique.